### DISCUSSION TITLE
Official Figma Library

### DISCUSSION THREAD

**Discussion by cheema-corellian** [adobe/spectrum-css#2667] [General]
I am new to Adobe Spectrum, so please excuse me if I am asking a question with an obvious answer.

My team is looking to adopt Adobe Spectrum for use in our React app.  I was looking to see if there was a design library that our designer could work with, and I noticed an unofficial one by Adi Chakravarty.  It looks great.  However, it is missing some components.  For example, it does not include the calendar component and coverage for table component is quite limited.

I was wondering if there is another Figma library that contains some of those missing components or if there is a plan to create one.

Thanks.

**Comment by castastrophe** [comment]
The design documentation is linked from our documentation site: https://spectrum.adobe.com.
